packages/ui/src/memberships/components/MemberStack/MemberStack.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { memo } from 'react'
+import styled from 'styled-components'
+
+import { DefaultTooltip, Tooltip, TooltipContainer } from '@/common/components/Tooltip'
+import { TextInlineSmall } from '@/common/components/typography'
+import { Colors } from '@/common/constants'
+import { spacing } from '@/common/utils/styles'
+import { MemberPhoto } from '@/memberships/components'
+import { MemberAvatar } from '@/memberships/components/Avatar'
+import { Member } from '@/memberships/types'
+
+export type MemberSumary = Pick<Member, 'id' | 'handle' | 'avatar'>
+
+interface MemberStackProps {
+  members: MemberSumary[]
+  max?: number
+}
+export const MemberStack = memo(({ members, max = 5 }: MemberStackProps) => {
+  const hasExtra = members.length > max
+  const toDisplay = hasExtra ? members.slice(0, max - 1) : members
+  const remaining = +hasExtra && members.length - max + 1
+
+  return (
+    <MemberStackStyles>
+      {toDisplay.map(({ id, handle, avatar }) => (
+        <Tooltip key={id} tooltipTitle={handle} tooltipText={`Worker ID: ${id}`}>
+          <MemberAvatar avatarUri={avatar} />
+        </Tooltip>
+      ))}
+      {hasExtra && (
+        <Tooltip tooltipText={`And ${remaining} more`}>
+          <HiddenMember>
+            <TextInlineSmall bold>+{remaining}</TextInlineSmall>
+          </HiddenMember>
+        </Tooltip>
+      )}
+    </MemberStackStyles>
+  )
+})
+
+const HiddenMember = styled(DefaultTooltip)`
+  background-color: ${Colors.Blue[50]};
+  border: 1px solid ${Colors.Black[200]};
+  box-sizing: content-box;
+  height: 26px;
+  width: 26px;
+
+  ${TextInlineSmall} {
+    font-size: 10px;
+  }
+`
+const borderWidth = '2px'
+const MemberStackStyles = styled.div`
+  display: flex;
+
+  ${TooltipContainer} {
+    margin-right: calc(${spacing(-1)} - ${borderWidth});
```

Comment:
```
margin-right: calc(${spacing(-1)} - ${borderWidth});
will-change: margin-right;
transition: ${Transitions.all};
```

and use box-shadow instead of border. It will make 24px size of circles, less calculating and better vertical alignment.

Reply:
@ClumsyVlad thanks for the review. I tried with box-shadow but the circle gets visibly out of shape. Also I noticed there is a problem with the tooltips position (they don't appear exactly under the circles). Is there an easy way to fix that ? The design/wiredframe doesn't define the hover interaction so maybe we should check with Natalia/Artur before spending more time on this. WDYT ?

Reply:
I'll ask Artur about this and will try to align this parts with/without :hover state. Now we can merge it as well, thank you.